autosubmit/platforms/paramiko_platform.py

```diff
@@ -435,24 +473,22 @@ def send_file(self, filename, check=True):
             self.delete_file(filename)
         try:
             local_path = os.path.join(os.path.join(self.tmp_path, filename))
-            remote_path = os.path.join(
-                self.get_files_path(), os.path.basename(filename))
+            remote_path = os.path.join(self.get_files_path(), os.path.basename(filename))
```

Comment:
No difference, just placed everything into one line as it fit the under ~120 limit